docs/src/simulation_tips.md

```diff
@@ -188,8 +189,7 @@ For large simulations on the GPU, careful management of memory allocation may be
   ```
 
 - Try to use higher-order advection schemes. In general when you use a higher-order scheme you need
-  fewer grid points to achieve the same accuracy that you would with a lower-order one. Oceananigans
-  provides two high-order advection schemes: 5th-order WENO method (WENO) and 3rd-order upwind.
+  fewer grid points to achieve the same accuracy that you would with a lower-order one.
```

Comment:
Do we have a page discussing the advection schemes that are available? If so, we should probably link it here

Reply:
I don’t think we do. There is this —> https://clima.github.io/OceananigansDocumentation/stable/appendix/library/#Advection

Reply:
I added a reference to the above. It's better than nothing atm.